.github/PULL_REQUEST_TEMPLATE/PR_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+Thank you for contributing. You're awesome!
+
+When you have submitted the request don't forget to ask for a review. We may respond with some clarifying questions or request minor changes before approval and merge.
+
+Please clearly fill out all the information below.
+
+### What
+##### In words what does this PR address (bug fix, feature, etc.)?
+
+### How
+##### Describe the current behavior, what the new behavior will be, and how you achieved your results.
```

Comment:
This is purely stylistic (so feel free to ignore it), but it seems a little odd to have headers like this. Maybe consolidate it into a single header that fully explains the thing.